docs/conf.py

```diff
@@ -10,7 +10,7 @@
 
 myst_enable_extensions = ["dollarmath"]
 
-html_logo = "_static/logo.png"
+html_logo = "_static/WAM_logo_v3.png"
```

Comment:
Old logo's are also available in git history, so we might as well keep the version out of the logo.